guides/examples/basic-web-app.md

```diff
@@ -164,11 +260,38 @@ ockam policy create --at web --resource tcp-inlet --expression '(= subject.compo
 ockam tcp-inlet create --at /node/web --from 127.0.0.1:5432 --to /project/default/service/forward_to_db/secure/api/service/outlet
 ```
 
-Take note of the `--from` and `--to` values above. The `--from` is telling the node to listen on port 5432, the default postgres port, and to forward it `--to` the forwarder service to the database that we created in the previous section. This means requests to localhost:5432 will be forwarded to whatever node has registered as `db`, wherever it is!
+Take note of the `--from` and `--to` values above. The `--from` is telling the node to
+listen on port `5432`, the default postgres port, and to forward it `--to` the forwarder
```

Comment:
Same thing here we should be talking about the `relay` service

Reply:
done